nicolas-martin-castellanos/Tarea-3/funciones/Last Survivor.js

```diff
@@ -0,0 +1,22 @@
+function largestPairSum(array) {
+  var longitud=array.length
+  console.log(longitud)
+  var aux=0
+
+   for(var i=0; i<longitud; i++) {
```

Comment:
@NicolasMartin23 , you should use modern approaches like, forEach, reduce or map